src/App.jsx

```diff
@@ -1,9 +1,10 @@
 import './App.css';
-
+import Sara from './components/Sara';
```

Comment:
can we make this dynamic imports 
```suggestion
import {Sara} from './components/Sara';
```

Reply:
sure , I will do that :)